src/test/java/org/jenkinsci/plugins/workflow/cps/DSLTest.java

```diff
@@ -604,6 +609,56 @@ public void namedSoleParamForStep() throws Exception {
         WorkflowRun b = r.assertBuildStatusSuccess(p.scheduleBuild2(0));
     }
 
+    @Test public void passwordParameter() throws Exception {
```

Comment:
I added `ArgumentsActionImpl.passwordParametersSanitized` in https://github.com/jenkinsci/workflow-cps-plugin/pull/370/commits/78111484e999fe68e87b7345d45ff68121c0d396 that is basically a subset of this test, so I would probably delete my old test so we don't test the same thing twice and since this test adds some assertions for the warnings that the old test didn't have so it is better to keep this one.